form-validation.js

```diff
@@ -7,29 +7,26 @@ const messageMobile = document.querySelector('#messageEmail-mobile');
 const nameInput = document.querySelector('.name');
 const emailInput = document.querySelector('.email');
 const messageInput = document.querySelector('.message');
-const rawData = localStorage.getItem('cachedFormData');
+const savedData = JSON.parse(localStorage.getItem('formData')) || {};
 
-const formobject = {
-  name: '',
-  email: '',
-  message: '',
-};
-
-if (rawData) {
-  const formobject = JSON.parse(rawData);
-  emailInput.value = formobject.email;
-  nameInput.value = formobject.name;
-  messageInput.value = formobject.message;
+if (savedData) {
+  nameInput.value = savedData.name;
+  emailInput.value = savedData.email;
+  messageInput.value = savedData.message;
```

Comment:
- [ ] As the previous reviewer mentioned too, please do not forget that users should have the superpower to retrieve data and have forms prefilled if it exist. Of course, it's working just fine on mobile screens. I just hope we can get this to work on bigger screens too. What do you think, @stino-x? :wink:

Reply:
@AbayomiOlaoye ill try to do it quickly, Thanks!!!

Reply:
@stino-x , I believe in your ability. Good luck with it. I am sure it will be approved this time, trust me :people_hugging:

Reply:
@AbayomiOlaoye i just made changes could you kindly check them out?

Reply:
Excellent! All checks well. Please submit again from your dashboard so that a colleague can pick it up :clap: Good job!